specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2018-06-01-preview/applications.json

```diff
@@ -1,403 +1,408 @@
-{

```

Comment:
Not quite sure what happened with the diff. The changes here were:
- Updated version
- Changed Applications_List to Applications_ListByCluster
- Changed description of Applications_ListByCluster
- Changed ApplicationGetProperties to ApplicationProperties
- Changed casing of Errors
- Added x-ms-parameter-location to parameters